.buildkite/pipeline.yml

```diff
@@ -107,3 +107,48 @@ steps:
         depends_on:
           - "operator-image-build"
           - "e2e-tests-image-build"
+
+  - group: redhat-release
+
+    steps:
+
+      - label: "Generate CRDs"
+        key: "generate-crds"
+        if: build.message == "run operatorhub release"
+        commands:
+           - make generate-crds-v1
+
+      - label: "run container push"
+        key: "redhat-container-push"
+        if: build.message == "run operatorhub release"
+        depends_on:
+          - "generate-crds"
+        commands:
+           - cd hack/operatorhub
+           - make build
+           - ./operatorhub container push --dry-run=false
+
+      - label: "run container preflight"
+        key: "redhat-preflight"
+        if: build.message == "run operatorhub release"
+        agents:
+          image: "quay.io/opdev/preflight:1.2.1-linux-amd64@sha256:6f63033a934d9a9821cea2247858b0557f471b1f0fc9c4f691ae1c934d44d90b"
```

Comment:
I wonder here if pulling the preflight binary while running with our default agent image (or adding the preflight binary to our agent) would be a better alternative than what's being done below.  Would love thoughts about this approach.